client/cli.rs

```diff
@@ -224,6 +226,25 @@ pub enum SubCommand {
         input: String,
     },
 
+    /// Send a UNIX signal to one or multiple tasks.
+    /// DISCLAIMER: This bypasses Pueue's process handling logic!
+    ///     You might enter weird invalid states, use at your own descretion.
+    #[cfg(not(target_os = "windows"))]
+    Signal {
```

Comment:
It is easy to implement the `-g` flag? i.e. send the signal to a group.

Reply:
I.e. sending the signal to all running tasks of a specific group?

Reply:
yep

Reply:
After some consideration, I went back and decided to move the signal handling to the `--signal, -s` flag to kill.

The logic is basically the same and we only need very few adjustments.
The help text on the flag should also clarify things and warn the user about their actions.

Reply:
Thanks! Yeah making the flag only work under POSIX systems also works.